sections/baseline_expenses.md

```diff
@@ -0,0 +1,126 @@
+# Baseline Expenses
+
+## Get all baseline expenses in a project
+
+- `GET /projects/{projectId}/baseline_expenses` - Returns all baseline expenses for a given project.
+
+| Response fields         | Description/format                         |
+| ----------------------- | ------------------------------------------ |
+| id                      | Integer                                    |
+| project_id              | Integer, ID of project                     |
```

Comment:
Get rid of the "_id" of id fields. I know we use different things, this the most used in the public API is without.
Maybe project_id is not even needed as it can only be fetched from the project id.